controller/workitem_blackbox_test.go

```diff
@@ -320,7 +320,7 @@ func createPagingTest(t *testing.T, controller *WorkitemController, repo *testsu
 		count := computeCount(totalCount, int(start), int(limit))
 		repo.ListReturns(makeWorkItems(count), uint64(totalCount), nil)
 		offset := strconv.Itoa(start)
-		_, response := test.ListWorkitemOK(t, context.Background(), nil, controller, nil, nil, nil, nil, nil, &limit, &offset)
+		_, response := test.ListWorkitemOK(t, context.Background(), nil, controller, nil, nil, nil, nil, nil, nil, &limit, &offset)
```

Comment:
Also wondering if maybe we could define the filter argument as something like; filter[*], or simply ignore fetching the filter from the Controller Context and rather get it dynamically from the Request instead and match [(.*)] to some 'Filter' method. I'll have a look at that.